hugo/content/contribute.md

```diff
@@ -3,83 +3,279 @@ title: Contribute
+- Build your own version of Mumble, see [Building Mumble](https://github.com/mumble-voip/mumble#building).
 
-**Ice methods**: We have a [generated documentation](http://mumble.sourceforge.net/slice/) for exposed Ice methods. Although probably only developers using the doc will be able to understand it, please give us a note on any inconsistencies, errors or needed clarifications if you find them.
+If you find any issues, take a look at [Report Bugs](#report-bugs-and-feature-requests).
 
-# Promotion
+<!-- mention documentation? Potential problem with TOC linking as the categories have the same name -->
+<!-- 
+### Documentation
+
+See [below](#documentation). -->
 
-Get the word out. This means more than writing about it on your blog ;)
+### Mumble theme
 
-* Ask (voice-)server hosters on if they also provide Mumble servers if you can’t find them.
-* Coordinate with one of the hosters providing free mumble servers and start promoting Mumble as the voicechat of choice for your favorite game.
-* Add mumble hosters you found to the {{< wiki Hosters />}} wiki page / update information.
-* Create How-Tos, Screencasts, Tutorials or Benchmarks on typical issues and topics, and publish them on prominent platforms.
-* Spread the word in groups – join our [Steam Group](https://steamcommunity.com/groups/Mumble_VoIP)
+<!-- Add more details, descriptions etc.? -->
+You find the Mumble theme in it's own [repository on Github](https://github.com/mumble-voip/mumble-theme).
 
-# Skinning
+#### Skins
 
 Mumble skins bring variety into Mumbles design and choices for different tastes.
 
 Create new skins for Mumble! Information can be found on our {{< wiki Skinning "Skinning wiki page" />}} and existing skins we know about on the {{< wiki Skins "Skins wiki page" />}}.
 
-# Coding
 
-## 3rd party Interfaces
+# Documentation & Website
 
-Web interfaces, web-viewers, administration programs, CMS website modules – all neat things we would love to see! These could be written in any language supported by our Ice based RPC Interface (using the deprecated {{< wiki DBus />}} Interface is no longer advised). You could start your own project or link up with an {{< wiki "3rd Party Applications" "already existing one" />}}. To get an idea about how interaction with Murmur might look like you can take a look at the example scripts in our git repositories script folder.
+Improve our Documentation and Website.
 
-## Ice authenticators
+## Documentation
 
-See {{< wiki Authenticators />}}. We’re happy for every authenticator and authenticator improvement you and the community can bring us!
+<!-- Todo: 
 
-## Positional Audio
+Add a concrete plan:
+- what documentation we want to have
+- what should be converted from the wiki
+- what is done & what is not
```

Comment:
This again is very prone to become outdated...

Reply:
Thats what Issues are there for.
We can have a todo list, that gets updated.
In this text, we only link to the issue.

Reply:
I think we should rather use issues and milestones/projects in the GitHub repo directly. And here we could link to the general milestone/project but not to specific issues :thinking:

Reply:
Sounds good.